auth/user_mgt.go

```diff
@@ -1298,6 +1304,11 @@ var serverError = map[string]*authError{
 		message:  "user with the provided email already exists",
 		authCode: emailAlreadyExists,
 	},
+	"EMAIL_NOT_FOUND": {
+		code:     internal.NotFound,
+		message:  "no user record found for the given email",
```

Comment:
Is this the error string shown in an error message?

In the other langs we made these messages "grammatical" with caps and punctuation, like "No user record found for the given email."

Reply:
I agree with you in general but it seems like all error messages follow this pattern in Go: 
https://github.com/firebase/firebase-admin-go/blob/4121c50b610f0ce408a2978cf14b306becf276d3/auth/user_mgt.go#L1280
Will need to confirm with @hiranya911

Reply:
Marking this as resolved as this is consistent with Go conventions: https://github.com/golang/go/wiki/CodeReviewComments#error-strings